tests/dynrpg.cpp

```diff
@@ -0,0 +1,204 @@
+#include <lcf/data.h>
+#include "doctest.h"
+#include "dynrpg.h"
+#include "game_variables.h"
+#include "test_mock_actor.h"
+
+TEST_SUITE_BEGIN("DynRPG");
+
+class LogGuard {
+public:
+	LogGuard() {
+		lvl = Output::GetLogLevel();
+		Output::SetLogLevel(LogLevel::Error);
+	}
+
+	~LogGuard() {
+		Output::SetLogLevel(lvl);
+	}
+
+	LogLevel lvl;
+};
```

Comment:
`MockActor` does this as well, might as well use it and not have to invent a new guard.

Reply:
Addressed your two comments. I still like my pun on LockGuard :P